### ISSUE TITLE
Baritone / Schematics / Build Command

### ISSUE THREAD

**Issue by 1Truedg** [cabaletta/baritone#889]
So I have more or less 3 questions which I hope didnt get answered already.

- Is there a plan to somehow support the build command/schematics in 1.14 or are there plans for 1.15 to somehow work with older builds due to I think changed block IDs in older schematics or so, not sure.

- Is there a way for Baritone to tell which blocks are needed + how much times for a build?

- Is it possible to pause/resume the build command to refill the blocks if you are in survival and let it resume from the old position? If not would the last 2 be a possible thing which would come in the near future or probably not?

**Comment by LoganDark** [comment]
> Is it possible to pause/resume the build command to refill the blocks if you are in survival and let it resume from the old position?

This will be possible when my fork gets merged into Baritone. Its pause/resume commands work with every process, build process, pathing process, follow process etc.

**Comment by Shy2Soul** [comment]
> Is there a way for Baritone to tell which blocks are needed + how much times for a build?

No, but if you use baritone/Impact on forge, [schematica](https://www.curseforge.com/minecraft/mc-mods/schematica) can tell you what you need.

**Comment by Fabianths** [comment]
> > Is it possible to pause/resume the build command to refill the blocks if you are in survival and let it resume from the old position?
> 
> This will be possible when my fork gets merged into Baritone. Its pause/resume commands work with every process, build process, pathing process, follow process etc.

When is it going to be merged

**Comment by LoganDark** [comment]
When I have enough time to address the review comments

**Comment by AxtonF** [comment]
Sorry to hijack this, but was your fork merged? And more importantly is it included in the newer 1.16 releases?

**Comment by LoganDark** [comment]
Probably.

**Comment by l1ving** [comment]
> And more importantly is it included in the newer 1.16 releases?

Probably not. I said I'd get around to it either this weekend or the next one

**Comment by l1ving** [comment]
Oh I misunderstood. Yea, pause / resume work on everything just fine
